recipe/meta.yaml

```
@@ -31,11 +31,11 @@ requirements:
     - wheel
   run:
     - importlib_resources >=5.10.0
-    - numpy >=1.22.0,<1.25.0
-    - pandas >=1.4.3
+    - numpy >=1.25.0
```

```suggestion
    - numpy >=1.25.0, <2.0.0
```
per line https://github.com/alteryx/woodwork/blob/e2f0adadd9371858f0838f264f3fd755040c8b31/pyproject.toml#L38

---

That's not correct. They removed the upper bound in >=0.29.0, see https://github.com/alteryx/woodwork/commit/a32ac361ce3f251bc5801a4bb3c43e3eff3ffff2

---

They added it back after the last release though. If you look at https://github.com/alteryx/woodwork/compare/v0.31.0...main

---

Thank you. Fair enough. I fixed it.